llvm/lib/Target/RISCV/GISel/RISCVLegalizerInfo.cpp

```diff
@@ -581,6 +581,12 @@ RISCVLegalizerInfo::RISCVLegalizerInfo(const RISCVSubtarget &ST)
 
   SplatActions.clampScalar(1, sXLen, sXLen);
 
+  getActionDefinitionsBuilder(G_INSERT_SUBVECTOR)
+      .legalIf(all(typeIsLegalBoolVec(0, BoolVecTys, ST),
```

Comment:
It should be `customIf` again.

Reply:
It is still legalIf.

Reply:
I don't know how my change didn't get added once again. Im sorry.